cred/src/main/java/org/opencadc/cred/CredInitAction.java

```diff
@@ -90,21 +97,20 @@ public class CredInitAction extends InitAction {
     private static final Logger log = Logger.getLogger(CredInitAction.class);
 
     private static final String CONFIG_FILE = "cred.properties";
-    private static final String DELEGATE_PROP  = "org.opencadc.cred.delegate.allowedUser";
-    private static final String PROXY_PROP     = "org.opencadc.cred.proxy.allowedUser";
-    private static final String MAX_VALID_PROP = "org.opencadc.cred.proxy.maxDaysValid";
-    
+    private static final String MAX_VALID_PROP = "org.opencadc.cred.maxDaysValid";
+    private static final String SUPERUSER = "org.opencadc.cred.superUser";
+    public static final File SIGN_CERT_FILE = new File("/config/signcert.pem");
```

Comment:
This should be loaded from {user.home}/config/{filename}; that way the code (war file) could work if deployed in a different environment (eg not using our standard practice supported by the cadc-tomcat base image).

{user.home} being the java system property

Reply:
Fixed